packages/design/src/Typography.mdx

```diff
@@ -0,0 +1,131 @@
+      free of flourish.
+    </Text>
+  </Content>
+</Playground>
+
+Heading and Text are extended by the ability to add
+[Emphasis](/packages-components-src-emphasis-emphasis) to these components as
+needed.
+
+There are some foundational elements that make up these components:
+
+## Font families
+
+### Display
+
+[Poppins](https://fonts.google.com/specimen/Poppins) is our display face across
+all of Jobber's platforms. It has a heft that makes it especially well-suited
+for high-level headings, and its' association with our brand makes it effective
+for labeling our [Buttons](/components/button).
+
+<Playground>
+  <Content>
+    <Heading level={2}>Poppins is used for bold statements</Heading>
+    <Button label="And punchy labels"></Button>
+  </Content>
+</Playground>
+
+### Base
+
+[Source Sans Pro](https://fonts.google.com/specimen/Source+Sans+Pro) is our most
+broadly-used typeface. It is used for body copy, most headings, data display,
+form inputs, labels, and more!
+
+<Playground>
+  <Content spacing="large">
+    <Heading level={3}>Source Sans Pro is for less prominent headings</Heading>
+    <Text>
+      You can also use it for longer, more readable statements in paragraphs,
+      descriptive text, in lists, and more. If you've been paying careful
+      attention, you might have noticed that Atlantis even uses Source Sans Pro
+      for presenting documentation!
+    </Text>
+    <Checkbox
+      checked
+      label="Source Sans Pro can label inputs nicely"
+    ></Checkbox>
+    <InputText
+      defaultValue="It's great for input values too"
+      placeholder="Source Sans Pro for labels"
+    />
+    <Table>
+      <Header>
+        <Cell>Source</Cell>
+        <Cell>Sans</Cell>
+        <Cell align="right">Pro</Cell>
+      </Header>
+      <Row>
+        <Cell>Data point</Cell>
+        <Cell>More data</Cell>
+        <Cell align="right">$280</Cell>
+      </Row>
+      <Row>
+        <Cell>Another data point</Cell>
+        <Cell>Additional values</Cell>
+        <Cell align="right">$200</Cell>
+      </Row>
+    </Table>
+  </Content>
+</Playground>
+
+## Font sizes
+
+Our base font-size is based on our [Spacing](/space) scale for a consistent
+sizing rhythm across Jobber. At its' default zoom level, the equivalent is
+`16px`. While this is higher than in some systems where `14` may be more common,
+Source Sans Pro is optically a bit smaller, particularly at the x-height, so we
+start with `16` and work our way up and down from there.
+
+## Line height
+
+Line height, or
+[leading](https://www.shillingtoneducation.com/blog/leading-typography/), is the
+space between subsequent lines of type. Our `Heading` and `Text` components
+handle their line-heights internally, so you don't need to worry specific
+implementation values as long as you're working with those components.
+
+Our base line-height of `1.25` is optimized for reading multiple lines of
```

Comment:
If possible, would it make sense to inherit the line height appropriate for the size of the font being used? My assumption is while we have options, there is a preferred choice per type size or font?

Just trying to think of ways to avoid minor variation if that choice is presented.

Reply:
`Heading` and `Text` will never vary in the line-heights that they present for each "level" (for both web and mobile) so I wanted to stay away from specific details here... maybe talking less about the values would be helpful? In Atlantis we actually intentionally decoupled line-height from font-size because of the reasons described later in this paragraph (at one point setting a font-size of `large` would give you a `large` line-height, which gets less useful as you get into larger font-sizes). I'm thinking I could get rid of this whole paragraph and update the previous one to say

> Line height, or [leading](https://www.shillingtoneducation.com/blog/leading-typography/), is the space between subsequent lines of type. The line-heights in the `Heading` and `Text` components are optimized for each variation's unique combination of font-size, weight, and family, so you don't need to adjust their line-height manually when you use them.

Reply:
Then the subsequent paragraph could read

> At smaller sizes, a more generous line-height relative to the font-size is used to allow the user to more easily parse multiple lines of small text. As font-size increases into "display" sizes such as a level 1 `Heading`, the ratio of line-height to font-size decreases as the larger font-size is more readable and requires less line-to-line breathing room.

Reply:
Yeah, I think that would be 👌 . The previous version almost promoted the customization of it, being clear that it's not required to adjust using this component is solid.